lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-cloud_manager-provision.rb

```diff
@@ -1,5 +1,5 @@
 module MiqAeMethodService
-  class MiqAeServiceMiqProvisionCloud < MiqAeServiceMiqProvision
+  class MiqAeServiceManageIQ_Providers_CloudManager_Provision < MiqAeServiceMiqProvision
```

Comment:
I'd like to... but that didn't work because of the service models' autoloads.

We should almost certainly end up doing something different: this is more of a "good enough for now" solution.